src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1743,44 +1200,59 @@ private class ACCT_IndividualAccounts_TEST {
     }
 
     /*********************************************************************************************************
-    * @description Update a household account's record type, verify system fields are unset.
-    */
+    @description 
+        Update a Household Account's Record Type to another Record Type 
+    verify:
+        The specific system fields should not be set.
+    **********************************************************************************************************/ 
     public static testMethod void updateHHAccountRecType() {
+        Boolean isPersonAccountEnabled = UTIL_Describe.isValidField('Account', 'isPersonAccount');
 
-        List<RecordType> rectypes = [SELECT Id FROM RecordType WHERE SobjectType = 'Account' AND IsActive = true];
-        if (rectypes.size() < 2) {
+        String soql = String.format(
+            'SELECT Id{0} FROM RecordType WHERE SobjectType = \'\'Account\'\' AND IsActive = true',
```

Comment:
Good question. I think it is easier to read/maintain/debug the query when it is one String where variations are implemented using parameters {0}, {1}, ...
On the other side, there were different discussions of what is better wrt performance and memory fragmentation for String concat using "+" vs String.format() vs StringBuilder. The concat is re-creating a new String for every append (it is overhead). The String.format() is using less fragmentation. But, the main reason is readability and maintenance.